static/css/cf-theme-overrides.less

```diff
@@ -121,6 +121,9 @@
 // .error
 @input-error:                   @redorange;
 
+// .warning
+@input-warning:                 @gold;
```

Comment:
Ohh wait, if this is being added here to cf-theme-overrides is this overriding an existing CF component, or a proposed component? (e.g. cf-notifications.) If it's overriding a component that is not yet part of CF, it should be removed here and left in cf-enhancements.